src/index.html

```diff
@@ -1,12 +1,328 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <img
+        class="logo"
+        src="../src/images/logo.png"
+        alt="Moyo logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a
+              class="nav__cell is-active"
+              href="#apple"
+              data-qa="card"
+            >Apple</a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__cell" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__items" >
+            <a class="nav__cell" href="#smartphone">Smartphones</a>
+          </li>
+          <li class="nav__items">
+            <a
+              data-qa="nav-hover"
+              id="shown"
+              class="nav__cell"
+              href="#laptop&computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__cell" href="#gadjets">Gadgets</a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__cell" href="#tablets">Tablets</a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__cell" href="#photo">Photo</a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__cell" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__main">
+      <section class="catalog">
+        <div class="card" data-qa="card">
+          <div class="card__items card__items--photo"></div>
+          <h1 class="card__items card__items--name"
+            >APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
```

Comment:
```suggestion
          <h2 class="card__items card__items--name">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
```
The h1 tag can be just one on the page